centml/compile.py

```diff
@@ -0,0 +1,98 @@
+import builtins
+from typing import Callable, Dict, List, Optional, Union
+
+import torch
+from prometheus_client import Gauge, start_http_server
+from torch._subclasses.fake_tensor import FakeTensorMode
+
+from centml.compiler.backend import centml_dynamo_backend
+from centml.compiler.config import OperationMode, settings
+from centml.compiler.profiler import Profiler
+
+start_http_server(8000)
+
+
+class Metric:
+    def __init__(self, gpu_name):
+        self._time = 0
+        self._metric = Gauge(f'{gpu_name}_metric', f'Sum of the kernel execution times on {gpu_name}')
```

Comment:
I feel like Histogram metric might be a better one to use. https://prometheus.github.io/client_python/instrumenting/histogram/